### ISSUE TITLE
Bug in the 1.100 with Button 9 and 21

### ISSUE THREAD

**Issue by Vtah** [danielfppps/hydrobuddy#15]
There is a bug in version 1.100: buttons 9 and 21 bring up the "Tissue Analysis" window. The "Tool Precession" window is not called. I fixed it for myself, but I can’t fix  your repository. Thanks for a good product!

**Comment by vlucent** [comment]
yeah, it's kind of impossible to set instrument precision values because the button triggers Tissue Analysis instead. Is there a workaround? @danielfppps

**Comment by vlucent** [comment]
okay, so it looks like in file hb_main.lfm, both Button9 and Button21 are linked to the procedure Button9Click in the OnClick parameter:
```
object Button9: TButton
    Left = 222
    Left = 224
    Height = 25
    Hint = 'Click this button to open up the instrument precision settings dialogue'
    Top = 142
    Top = 110
    Width = 246
    Caption = 'Set Instrument Precision Values'
    OnClick = Button9Click
    TabOrder = 5
end
```

```diff 
+ object Button21: TButton
+    Left = 224
+    Height = 25
+    Hint = 'Click this button to open up the instrument precision settings dialogue'
+    Top = 142
+    Width = 246
+    Caption = 'Tissue Analysis'
+    OnClick = Button9Click
+    TabOrder = 9
+ end
```

while Button9Click was repurposed for tissue analysis and the insprecision function deleted and not reimplemented elsewhere.

```diff
   procedure TForm1.Button9Click(Sender: TObject);
   begin
-    hb_insprecision.Form7.Visible := True;
+    hb_tissue_analysis.Form16.UpdateTissueList;
+    hb_tissue_analysis.Form16.Visible := True;
   end;
```

So, the solution would probably be to make a Button21Click procedure dedicated for tissue analysis and restore Button9Click's original function.

**Comment by Vtah** [comment]
Need to change HB_main.pas:
procedure TForm1.Button9Click(Sender: TObject);
begin
  Form7.Show;
end;  
procedure TForm1.Button21Click(Sender: TObject);
begin
  hb_tissue_analysis.Form16.UpdateTissueList;
  hb_tissue_analysis.Form16.Visible := True;
end;

**Comment by ghost** [comment]
I have the fix @vlucent @Vtah but Github is so damn confusing that I made of mess of things in my fork trying to send @danielfppps pull requests. I am new to Github not thrilled with it so far.
